pkgs/development/perl-modules/Bio-BigFile/default.nix

```
@@ -0,0 +1,75 @@
+{ lib, perl, buildPerlModule, perlPackages, fetchurl, fetchFromGitHub, pkgs }:
+# This version of bio::DB::bigfile is suited for ENSEMBL VEP
+# See https://www.ensembl.org/info/docs/tools/vep/script/vep_download.html#bigfile
+
+# Even though there is a kent package on nix, we have to override it to an older version
+let
+  kent335 = pkgs.kent.overrideAttrs (old: rec {
+    version = "335";
+    name = "kent" ;
+
+    src = fetchFromGitHub {
+      owner = "ucscGenomeBrowser";
+      repo = name;
+      rev = "v${version}_base";
+      sha256 = "1455dwzpaq4hyhcqj3fpwgq5a39kp46qarfbr6ms6l2lz583r083";
+    };
+    # Use the ENSEMBL instructions, with some changes from
+    # https://github.com/NixOS/nixpkgs/blob/nixos-22.05/pkgs/applications/science/biology/kent/default.nix#L70
+    # We *cannot* use the package from nixpkgs (tests fails)
+    buildPhase =
+      ''
+      export MACHTYPE=$(uname -m)
+      export CFLAGS="-fPIC"
```

```suggestion
    buildPhase = ''
      export MACHTYPE=$(uname -m)
      export CFLAGS="-fPIC"
```
I am not sure if uname -m is pure.